### PR TITLE
Delete security context pods after jobs finish

### PR DESCRIPTION
**Pull request by mphammer**

### CODE REVIEW

File: examples/synopsysctl/securityContext.json [hunk 1]
```diff
@@ -1,61 +1,70 @@
...
-        "runAsUser": 1000,
-        "runAsGroup": 1000
+        "runAsUser": 1234,
+        "runAsGroup": 1234
     },
     "blackduck-nginx": {
-        "fsGroup": 1000,
-        "runAsUser": 1000,
-        "runAsGroup": 1000
+        "fsGroup": 1234,
```
Comment (msenmurugan) [inline examples/synopsysctl/securityContext.json]: Can we keep the numbers same as before? i.e 1000

File: pkg/synopsysctl/cmd_update.go [hunk 2]
```diff
@@ -581,6 +581,12 @@ func setBlackDuckFileOwnershipJob(namespace string, name string, pvcName string,
 		},
 		Spec: batchv1.JobSpec{
 			Template: corev1.PodTemplateSpec{
+				ObjectMeta: metav1.ObjectMeta{
+					Labels: map[string]string{
+						"app":       "blackduck",
+						"component": "job-pod",
```
Comment (msenmurugan) [inline pkg/synopsysctl/cmd_update.go]: can we have the component name as "ownership" or "change-ownership" or something meaningful?

File: examples/synopsysctl/securityContext.json (hunk 1 above)
Comment (mphammer) [reply examples/synopsysctl/securityContext.json]: Done

File: pkg/synopsysctl/cmd_update.go (hunk 2 above)
Comment (mphammer) [reply pkg/synopsysctl/cmd_update.go]: Actually the labels are not needed (I forgot to remove them). I just removed the labels

Comment (msenmurugan) [reply pkg/synopsysctl/cmd_update.go]: :-)

File: examples/synopsysctl/securityContext.json [hunk 3]
```diff
@@ -4,6 +4,15 @@
         "runAsUser": 1000,
         "runAsGroup": 1000
     },
+    "blackduck-postgres": {
+        "fsGroup": 26,
+        "runAsUser": 26,
+        "runAsGroup": 26
+    },
+    "blackduck-init": {
```
Comment (msenmurugan) [inline examples/synopsysctl/securityContext.json:12]: Still we need to have different security context for blackduck-postgres and blackduck-init?

File: pkg/synopsysctl/cmd_update.go [hunk 4]
```diff
@@ -624,11 +624,29 @@ func setBlackDuckFileOwnershipJob(namespace string, name string, pvcName string,
 			}
 			if job.Status.Succeeded > 0 {
 				log.Infof("successfully set the group ownership of files for PV '%s' in namespace '%s'", pvcName, namespace)
-				kubeClient.BatchV1().Jobs(job.Namespace).Delete(job.Name, &metav1.DeleteOptions{})
+
+				// Delete the Job
+				err = kubeClient.BatchV1().Jobs(job.Namespace).Delete(job.Name, &metav1.DeleteOptions{})
```
Comment (msenmurugan) [inline pkg/synopsysctl/cmd_update.go]: Can you please set the delete options to have "propagationPolicy": "Background" and see the dependent pods are deleted?

Comment (msenmurugan) [reply pkg/synopsysctl/cmd_update.go]: so that you don't need the following codes.

**Review by msenmurugan — APPROVED** [review]
/lgtm

**Pull request merged**